### ISSUE TITLE
williams county: file name vs alt text

### ISSUE THREAD

**Issue by kevindherman** [proudcity/wp-proudcity#1773]
@aschmoe the hover over is displaying the image file name instead of the alt text:

https://williams-county-nd.proudcity.com/departments/human-resources/

This is one example but is the case throughout the majority of the site.

**Comment by aschmoe** [comment]
@kevindherman https://www.24a11y.com/2017/the-trials-and-tribulations-of-the-title-attribute/

Basically 'title' is not a good substitute for alt.

Options by order of accessibility preference:

1. Remove the title portion from the media
2. If they wish something to happen when they hover over the image, duplicate the alt text into the title attribute

**Comment by kevindherman** [comment]
@aschmoe if they leave the title blank will the hover default to the alt text?

**Comment by kevindherman** [comment]
@aschmoe the hover over text doesn't need to exist for the end user but if it does, it should be the alt text and not the title. 

Title in the media library is helpful since it is populated from the file name so it's helpful when searching as an admin.
